src/views/Header.js

```diff
@@ -80,6 +81,17 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
     ).options.headerTruncatedBackTitle;
   }
 
+  _getBackButtonTitleStyle(scene: NavigationScene): ?Style {
+    const lastScene = this._getLastScene(scene);
+    const { headerBackTitleStyle } = this.props.getScreenDetails(scene).options;
+    if (!lastScene) {
+      return headerBackTitleStyle;
+    }
+    const { headerTitleStyle } = this.props.getScreenDetails(lastScene).options;
+
+    return [headerTitleStyle, headerBackTitleStyle];
```

Comment:
I haven't used many iOS apps to say how the transition is supposed to happen, but I know that lots of apps have different styling for back button and title. A common example is how the title color is black and the back button has accent color in many apps.

To have a seamless transition, you need to ensure that all your titles and back button title have similar styles, if you don't, either
1. The back button in the current screen will look inconsistent with rest of the items
2. During the transition, two titles with different styles will cross-fade

Now, when you actually have two different styles between screens, and want to have a seamless transition, merging doesn't help, since you'll want to animate previous styles to current styles, not overwrite them.

@grabbou what does normal iOS apps do when 2 different screens have different styles? or is this very uncommon?